<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPasswordDialog</class>
 <widget class="QDialog" name="SetPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set your password</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="body"/>
   </item>
   <item>
    <layout class="QGridLayout" name="pswdsLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="alignment">
        <set>Qt::AlignLeft</set>
       </property>
       <property name="text">
        <string>Confirm:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="alignment">
        <set>Qt::AlignLeft</set>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="PasswordEdit" name="repasswordlineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="PasswordEdit" name="passwordlineEdit"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QProgressBar" name="passStrengthMeter">
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>Password strength: %p%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordEdit</class>
   <extends>QLineEdit</extends>
   <header>src/widget/passwordedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>passwordlineEdit</tabstop>
  <tabstop>repasswordlineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
